### DISCUSSION TITLE
Can more CPU cores be utilised?

### DISCUSSION THREAD

**Discussion by jmio23** [ggml-org/llama.cpp#391] [General]
When running this, it only ever uses 1 CPU core (on my intel MacBook pro), wondering if this is by design or some limitation that can't be avoided? or maybe it doesn't matter as much as I imagine?

I currently get about 1.6-2 words a second, I'm amazed to get even that tbh, but wondering if it can increased a lot further by utilizing more cores?

Maybe these is already a switch I am over looking?

**Comment by gjmulder** [comment]
`-t 2`
